varname.py

```diff
@@ -10,31 +10,44 @@
 
 import executing
 
-__version__ = "0.5.3"
+__version__ = "0.5.4"
 __all__ = [
     "VarnameRetrievingError", "varname", "will",
     "inject", "nameof", "namedtuple", "Wrapper", "debug"
 ]
 
+StringOrTuple = Union[str, Tuple[str]]
+
 class VarnameRetrievingError(Exception):
     """When failed to retrieve the varname"""
 
-def varname(caller: int = 1, raise_exc: bool = True) -> Optional[str]:
+def varname(
+        caller: int = 1,
+        multi_vars: bool = False,
+        raise_exc: bool = True
+) -> Optional[Union[StringOrTuple, Tuple["StringOrTuple"]]]:
```

Comment:
@alexmojaki  Like this? Not tested in mypy. And we didn't mention anything about mypy in the docs. Not sure if we need to test the whole package in mypy. Maybe in the future.

Reply:
No, not like that at all. Will discuss further tomorrow.